frontend/13f-app/src/App.js

```diff
@@ -15,6 +18,8 @@ function App () {
         <Route path='/' exact component= {Home} />
         <Route path='/search' exact component= {SearchResults} />
         <Route path='/company/:companyId' exact component= {SearchResults} />
+        <Route path='/company/:companyId/edgarfiling/' exact component= {FilingsPage} />
+        <Route path='/company/:companyId/edgarfiling/:filingId/data/' exact component= {DataPage} />
```

Comment:
Personally, it is still weird to me to have the path be so RESTy. This URL is visible to users so it's usually made a little more user-friendly. Even just dropping the last slash would help.